adventurous-anteaters/adv_ant/retroNet/templates/login.html

```diff
@@ -0,0 +1,34 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<div class="window-body" style="background-color: #dfdfdf;margin-left: 10px;margin-right: 10px;font-family:Pixelated MS Sans Serif;">
```

Comment:
This appears to be the same style as I commented on earlier, duplicated into another template. This is why classes are nice, so you can avoid this kind of duplication. If you wanted to change the background color on all of these, you'd have to update a whole bunch of duplicated code.